lib/amplitude/client.rb

```diff
@@ -0,0 +1,54 @@
+module AmplitudeAnalytics
+  # Amplitude
+  class Amplitude
+    attr_reader :configuration, :timeline
```

Comment:
is this public to the caller? if so, do the other analytics sdks also expose these fields?

Reply:
Here, `timeline` was exposed for testing purposes, `configuration` is exposed so users can change the config of the client, as in the [Python](https://www.docs.developers.amplitude.com/data/sdks/python/#configuration) Analytics SDK

Reply:
@Mercy811 I see, thanks for the explanation! I will have a look through exposing class attributes for the analytics package

Reply:
The `timeline` attribute in `Amplitude` is now hidden; `configuration` is still exposed as it is used by other classes